gerrit-entrypoint.sh

```diff
@@ -83,6 +83,8 @@ EOF
 done
 
   #Customize gerrit.config
+  set_gerrit_config repository."*".ownerGroup "Registered Users"
+
```

Comment:
To provide one way to do the setup is not a bad idea. However, I'd like to keep this option more open for those who don't want "Registered Users" as the default owner of new projects.